assemblers/org.osbuild.qemu

```diff
@@ -153,33 +157,40 @@ def mount(source, dest):
 
 def mkfs_ext4(device, uuid, label):
     opts = []
+    if uuid:
```

Comment:
A random one will be [generated](https://github.com/tytso/e2fsprogs/blob/master/misc/mke2fs.c#L3069). So for clients the best would be to specify both if you want 100% reproducible output.

Reply:
Hm, I think, as a general rule, we should not support random stuff being generated in the stages if we can avoid it (if the user doesn't care, they can randomly generate it themselves, but then it will at least occur in the manifest). So I would make the `uuid` a mandatory option, I think.

Reply:
Ok done.